terraform/alerting/slos.tf

```diff
@@ -14,59 +14,71 @@
 
 locals {
   default_per_service_slo = {
+    enable_alert      = false
     availability_goal = 0.995
-    latency = {
-      goal      = 0.9
-      threshold = 60000
-    }
-    enable_alert = false
+    latency_goal      = 0.99
+    latency_threshold = 60000
```

Comment:
Can you add in a comment the unit of this latency?

Reply:
Done.